yarn/pom.xml

```diff
@@ -109,6 +109,16 @@
       <properties>
         <jersey.version>1.9</jersey.version>
       </properties>
+      <!-- Enable hadoop-2.2 profile when the Hadoop version is 2.2.0, which is now 
+      the default. Those building against 2.2.x or some distro-specific variation of 
+      that string would still have to manually enable -Phadoop-2.2.
+      -->
+      <activation>
+       <property>
+        <name>hadoop.version</name>
```

Comment:
Can we just fold this config into the main POM config for yarn? 2.2 is the default after all, and I don't think YARN support works at all with Hadoop 1 at this point anyway. I could be missing something.